orchagent/isolationgrouporch.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2019 Broadcom.  The term "Broadcom" refers to Broadcom Inc.
+ * and/or its subsidiaries.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ISOLATIONGROUPORCH_H__
+#define __ISOLATIONGROUPORCH_H__
+
+#include "orch.h"
+#include "port.h"
+#include "observer.h"
+
+#define ISOLATION_GRP_DESCRIPTION       "DESCRIPTION"
+#define ISOLATION_GRP_TYPE              "TYPE"
+#define ISOLATION_GRP_PORTS             "PORTS"
+#define ISOLATION_GRP_MEMBERS           "MEMBERS"
+#define ISOLATION_GRP_TYPE_PORT         "port"
```

Comment:
From these defines its not obvious that ISOLATION_GRP_TYPE_PORT and ISOLATION_GRP_TYPE_BRIDGE_PORT define attributes values but all others define attributes type.

Reply:
@akokhan Thanks,  I have added additional description to to convey the difference.